elq.sh

```diff
@@ -1 +1 @@
-QMAIL/bin/maildir2mbox && exec elm ${1+"$@"}
```

Comment:
Don't touch elq, it will die right after 1.08. Same for pinq and qail.

Reply:
I applied the change to all except this one.